lib/concurrent_guest_installations.pm

```diff
@@ -112,6 +112,8 @@ sub install_guest_instances {
         else {
             $guest_instances{$_}->guest_installation_run(@_);
         }
+        # abort current guest installation if dry run failed
+        next if $guest_instances{$_}->{guest_installation_result} eq 'FAILED';
```

Comment:
This is not needed here, because there is already the following lines before this line:
```
if ($guest_instances{$_}->{guest_installation_result} ne '') {
    next;
}
```

Reply:
the result of dry run if after its else statement:
```
112         else {
113             $guest_instances{$_}->guest_installation_run(@_);
114         }

```
fg. https://openqa.opensuse.org/tests/3339451#step/unified_guest_installation/440

As the dry run already fails, the test need not attach or detach guest's screen.